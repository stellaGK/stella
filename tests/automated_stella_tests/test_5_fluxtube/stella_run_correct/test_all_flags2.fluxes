       #time        pflx                    vflx                    qflx
    0.0000E+000   -1.3156E-023   -5.6660E-023            NaN            NaN   -1.5515E-022    1.6439E-022
    1.0000E-002   -1.3390E-010   -1.3390E-010            NaN            NaN   -7.4861E-010   -6.8339E-010
    2.0000E-002    2.6432E-011    2.6432E-011            NaN            NaN    1.3712E-010    1.1720E-010
    3.0000E-002   -6.2792E-011   -6.2792E-011            NaN            NaN   -3.4429E-010   -2.7672E-010
    4.0000E-002    2.4488E-011    2.4488E-011            NaN            NaN    1.3454E-010    7.9628E-011
    5.0000E-002   -3.1700E-011   -3.1700E-011            NaN            NaN   -1.7486E-010   -1.0778E-010
    6.0000E-002    1.7079E-011    1.7078E-011            NaN            NaN    9.5027E-011    3.9297E-011
    7.0000E-002   -2.9390E-011   -2.9390E-011            NaN            NaN   -1.6379E-010   -9.1425E-011
    8.0000E-002    1.1573E-011    1.1573E-011            NaN            NaN    6.4161E-011    3.4459E-011
    9.0000E-002   -2.0149E-011   -2.0149E-011            NaN            NaN   -1.1188E-010   -8.4403E-011
    1.0000E-001    9.5316E-012    9.5315E-012            NaN            NaN    5.2503E-011    3.4608E-011
