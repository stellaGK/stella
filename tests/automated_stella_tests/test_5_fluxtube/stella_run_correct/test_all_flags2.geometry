#          rhoc          qinp          shat        rhotor          aref          bref     dxdXcoord      dydalpha    exb_nonlin  exb_nonlin_p
#    0.7107E+00    0.1111E+01   -0.8277E-01    0.7107E+00    0.4944E+00    0.2604E+01    0.1407E+01    0.7107E+00    0.5000E+00    0.0000E+00

     # alpha         zed        zeta        bmag bdot_grad_z        gds2       gds21       gds22       gds23       gds24     gbdrift     cvdrift    gbdrift0   bmag_psi0
  0.5000E+00 -0.3142E+01 -0.2642E+01  0.1123E+01  0.8488E-01  0.3451E+01 -0.8030E-02  0.2522E-02  0.8682E-01  0.1810E-01 -0.1974E+00 -0.1224E+00  0.1354E-01  0.1123E+01
  0.5000E+00 -0.2618E+01 -0.2118E+01  0.1070E+01  0.8455E-01  0.1116E+01  0.5575E-01  0.9813E-02  0.1556E-01 -0.5639E-01 -0.2939E+00 -0.2114E+00  0.9535E-02  0.1070E+01
  0.5000E+00 -0.2094E+01 -0.1594E+01  0.9776E+00  0.8625E-01  0.1066E+01 -0.9729E-01  0.1502E-01 -0.3740E-02  0.4425E-01  0.6127E-01  0.1601E+00 -0.3245E-02  0.9776E+00
  0.5000E+00 -0.1571E+01 -0.1071E+01  0.1102E+01  0.8004E-01  0.2663E+01 -0.1874E+00  0.1632E-01  0.5404E-01  0.4200E-01 -0.1359E+00 -0.5819E-01  0.2098E-01  0.1102E+01
  0.5000E+00 -0.1047E+01 -0.5472E+00  0.9822E+00  0.8616E-01  0.2052E+01  0.5092E-01  0.4484E-02 -0.1411E-01 -0.1013E-02 -0.6118E-01  0.3675E-01 -0.3613E-03  0.9822E+00
  0.5000E+00 -0.5236E+00 -0.2360E-01  0.1110E+01  0.7907E-01  0.1441E+01  0.1719E+00  0.2637E-01 -0.4127E-01  0.7370E-01  0.3756E-03  0.7703E-01 -0.1423E-01  0.1110E+01
  0.5000E+00  0.0000E+00  0.5000E+00  0.9564E+00  0.8880E-01  0.5184E+00  0.4242E-01  0.1556E-01  0.3308E-01 -0.6560E-02 -0.3169E-01  0.7160E-01  0.1458E-01  0.9564E+00
  0.5000E+00  0.5236E+00  0.1024E+01  0.1171E+01  0.8169E-01  0.3622E+01 -0.7056E-01  0.3965E-02 -0.6594E-01 -0.2594E-01 -0.3119E+00 -0.2430E+00 -0.1276E-01  0.1171E+01
  0.5000E+00  0.1047E+01  0.1547E+01  0.1015E+01  0.8944E-01  0.2219E+01  0.3796E-01  0.3834E-02 -0.5212E-01  0.4577E-01 -0.5830E-01  0.3332E-01 -0.1005E-01  0.1015E+01
  0.5000E+00  0.1571E+01  0.2071E+01  0.1005E+01  0.8890E-01  0.1130E+01  0.5426E-01  0.8735E-02  0.8246E-01 -0.3209E-01 -0.1295E+00 -0.3610E-01  0.1325E-01  0.1005E+01
  0.5000E+00  0.2094E+01  0.2594E+01  0.1215E+01  0.8163E-01  0.9435E+00 -0.9565E-01  0.2041E-01 -0.5002E-01 -0.7455E-01 -0.2858E+00 -0.2218E+00 -0.9634E-02  0.1215E+01
  0.5000E+00  0.2618E+01  0.3118E+01  0.9196E+00  0.9786E-01  0.1216E+01  0.1261E-01  0.4897E-02 -0.3741E-01  0.5920E-01  0.1814E-02  0.1135E+00 -0.1961E-02  0.9196E+00
  0.5000E+00  0.3142E+01  0.3642E+01  0.1161E+01  0.8384E-01  0.3509E+01  0.1395E+00  0.8179E-02  0.9596E-01 -0.3020E-01 -0.1862E+00 -0.1161E+00  0.1079E-01  0.1161E+01

