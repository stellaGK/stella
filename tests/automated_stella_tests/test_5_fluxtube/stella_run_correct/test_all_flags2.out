 istep=      0 time=  0.0000E+00  |phi|^2=  0.1309E-03 |apar|^2=   0.0000E+00  |bpar|^2=   0.0000E+00
 istep=      1 time=  0.1000E-01  |phi|^2=  0.4064E-04 |apar|^2=   0.0000E+00  |bpar|^2=   0.0000E+00
 istep=      2 time=  0.2000E-01  |phi|^2=  0.2073E-04 |apar|^2=   0.0000E+00  |bpar|^2=   0.0000E+00
 istep=      3 time=  0.3000E-01  |phi|^2=  0.1941E-04 |apar|^2=   0.0000E+00  |bpar|^2=   0.0000E+00
 istep=      4 time=  0.4000E-01  |phi|^2=  0.1292E-04 |apar|^2=   0.0000E+00  |bpar|^2=   0.0000E+00
 istep=      5 time=  0.5000E-01  |phi|^2=  0.8705E-05 |apar|^2=   0.0000E+00  |bpar|^2=   0.0000E+00
 istep=      6 time=  0.6000E-01  |phi|^2=  0.7213E-05 |apar|^2=   0.0000E+00  |bpar|^2=   0.0000E+00
 istep=      7 time=  0.7000E-01  |phi|^2=  0.7722E-05 |apar|^2=   0.0000E+00  |bpar|^2=   0.0000E+00
 istep=      8 time=  0.8000E-01  |phi|^2=  0.6408E-05 |apar|^2=   0.0000E+00  |bpar|^2=   0.0000E+00
 istep=      9 time=  0.9000E-01  |phi|^2=  0.4891E-05 |apar|^2=   0.0000E+00  |bpar|^2=   0.0000E+00
 istep=     10 time=  0.1000E+00  |phi|^2=  0.4275E-05 |apar|^2=   0.0000E+00  |bpar|^2=   0.0000E+00
